overlord/snapstate/handlers.go

```diff
@@ -3633,6 +3804,12 @@ func (m *SnapManager) undoRefreshAliases(t *state.Task, _ *tomb.Tomb) error {
 		}
 	}
 
+	aliasesPending := snapst.AliasesPending
+	if err := t.Get("old-aliases-pending", &aliasesPending); err != nil && !errors.Is(err, state.ErrNoState) {
```

Comment:
this probably needs a comment explanation why we do it this way

Reply:
it made me think, I simplified this part. thank you!